doc/source/whatsnew/v0.25.0.rst

```diff
@@ -33,6 +33,9 @@ Backwards incompatible API changes
 
 .. _whatsnew_0250.api_breaking.utc_offset_indexing:
 
+- Rich comparisons involving :class:`~Timestamp.Timestamp` return ``NotImplemented`` instead of raising ``TypeError`` with unsupported objects (:issue:`24011`)
```

Comment:
Can you revise this. I am not sure what you are actually changing.

Reply:
I proposed:

> Comparing :class:`~Timestamp.Timestamp` with unsupported objects now returns ``NotImplemented`` instead of raising ``TypeError``. This implies that unsupported rich comparisons are delegated to the other object, and is now consistent with Python 3 behavior for ``datetime`` objects (:issue:`24011`)

 I don't know exactly what is expected in term of details here.